src/main/java/sk/tomsik68/mclauncher/impl/login/yggdrasil/YDLoginService.java

```diff
@@ -22,12 +22,34 @@
 
 public final class YDLoginService implements ILoginService {
     public static UUID clientToken = UUID.randomUUID();
-    private static final String PASSWORD_LOGIN_URL = "https://authserver.mojang.com/authenticate";
-    private static final String SESSION_LOGIN_URL = "https://authserver.mojang.com/refresh";
-    private static final String SESSION_LOGOUT_URL = "https://authserver.mojang.com/invalidate";
+    private static String PASSWORD_LOGIN_URL = "https://authserver.mojang.com/authenticate";
```

Comment:
Is there a reason to leave these fields as `static` AND set them in constructors? Imagine you want to create more `YDLoginService` instances with each one using a different url. Leaving these fields static makes it impossible because all instances must use the static fields.
The same applies to `SESSION_LOGIN_URL` and `SESSION_LOGOUT_URL`.

Reply:
Very good point! Thank you)

Reply:
Stupid bag)